README.md

```diff
@@ -31,14 +40,14 @@ as provided by [Frab][frab-github]. The file format produced by the predecessor
 * In general it is possible to re-deployed the app for other events which
 provide the same event file format.
 
+## Build instructions
 
-## Release build / signing
-
-Before you can run a release build for your product flavor:
+In order to build this project the two build types can be configured as follows:
 
-1. Copy [`app/gradle.properties.example`](app/gradle.properties.example) to `gradle.properties`.
-2. Edit the required settings in `gradle.properties`.
+    Before importing the project in Android Studio
 
+    1. Create a file `gradle.properties` under directory `app`.
+    2. Copy the contents from the file [`app/gradle.properties.example`](app/gradle.properties.example) to `app/gradle.properties`.
```

Comment:
> In order to build this project the two build types can be configured as follows:

I would either explicitly mention `develop` and `release` or leave them out as they are already defined in the build.gradle. The decision more or less relates to the amount of knowledge one expects from someone interested in this project.

I find copying the content of a file somewhat error-prone. Therefore, I would rather recommend to make a copy. Here is a rough formulation - feel free to rewrite it.

In the `app` folder make a copy of the `gradle.properties.example` file and name it `gradle.properties`.
The file contains signing information which are used in build process.

I would not use the code formatting style (indentation) - this should be reserved for code snippets.

Reply:
I removed this line

> In order to build this project the two build types can be configured as follows: